vue/sbc-common-components/src/components/SbcSignin.vue

```diff
@@ -67,23 +67,26 @@ export default class SbcSignin extends NavigationMixin {
           // sync the account if there is one
           await this.syncAccount()
 
-          const currentUser = await this.getCurrentUserProfile()
-
           // redirect to create account page if the user has no 'account holder' role
           const isRedirectToCreateAccount = (userInfo.roles.includes(Role.PublicUser) && !userInfo.roles.includes(Role.AccountHolder))
```

Comment:
there is some logic in auth-web related to isRedirectToCreateAccount rite?Do we stil need that?

Reply:
wont need that, I will be removing that

Reply:
cool..